mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1644,6 +1645,17 @@ def _log_eval_table(self):
         data = data.assign(**columns)
         artifact_file_name = f"{metric_prefix}{_EVAL_TABLE_FILE_NAME}"
         mlflow.log_table(data, artifact_file=artifact_file_name)
+        if self.eval_results_path:
```

Comment:
@prithvikannan Can we add tests to this PR as well?

Reply:
Added!